### DISCUSSION TITLE
Icons in tabs

### DISCUSSION THREAD

**Discussion by dejury** [filamentphp/filament#3053] [Ideas]
Wouldn't it be nice to have a possibility to prepend the Tab label with an icon?
This would make it visually more attractive to see what each tab does.

```php
Tab::make('Settings')
        ->icon('heroicon-o-cog')
        ->schema([]);
```

At least for Form tabs, maybe more?

I am willing to create a PR for this, if there is enough support for this idea :)

**Comment by danharrin** [comment]
Yeah PR would be great. This is similar to wizard steps, which also have icons.

**Comment by dejury** [comment]
PR submitted, see: https://github.com/filamentphp/filament/pull/3058
